src/javascript/SelectorTypes/RichText/RichText.jsx

```diff
@@ -53,25 +49,32 @@ export const RichText = ({field, id, value, onChange, onBlur}) => {
         return <LoaderOverlay/>;
     }
 
-    const toolbar = loadOption(field.selectorOptions, 'ckeditor.toolbar');
-    const customConfig = loadOption(field.selectorOptions, 'ckeditor.customConfig');
+    const definitionConfig = {};
+    const CK_EDITOR_PREFIX = 'ckeditor.';
+    field.selectorOptions
+        .filter(option => option.name.startsWith(CK_EDITOR_PREFIX))
+        .forEach(option => {
+            definitionConfig[option.name.substring(CK_EDITOR_PREFIX.length)] = option.value;
+        });
+
+    // Resolve Toolbar
+    definitionConfig.toolbar = definitionConfig.toolbar || data.forms.ckeditorToolbar;
 
     // Delete the config set by GWTInitializer, as it may be wrong (Wrong site detection into DX, we will set it manually after)
     if (window.contextJsParameters && window.contextJsParameters.ckeCfg) {
         delete window.contextJsParameters.ckeCfg;
     }
 
-    let ckeditorCustomConfig = '';
-    if (customConfig && customConfig.value) {
+    if (definitionConfig.customConfig) {
         // Custom config from CND
-        ckeditorCustomConfig = customConfig.value.replace('$context', window.contextJsParameters.contextPath);
+        definitionConfig.customConfig = definitionConfig.customConfig.replace('$context', window.contextJsParameters.contextPath);
     } else if (data.forms.ckeditorConfigPath) {
-        ckeditorCustomConfig = data.forms.ckeditorConfigPath.replace('$context', window.contextJsParameters.contextPath);
+        definitionConfig.customConfig = data.forms.ckeditorConfigPath.replace('$context', window.contextJsParameters.contextPath);
 
         // Custom config coming from the template set, let's populate the contextJsParameters for retro compatibility
-        if (ckeditorCustomConfig) {
+        if (definitionConfig.customConfig) {
```

Comment:
We can group the "if" here:      
 if (definitionConfig.customConfig && window.contextJsParameters) {
     window.contextJsParameters.ckeCfg = definitionConfig.customConfig;
}